rest/src/main/java/com/ericsson/bss/cassandra/ecchronos/rest/RepairManagementRESTImpl.java

```diff
@@ -34,26 +38,38 @@
 import com.ericsson.bss.cassandra.ecchronos.core.repair.types.TableRepairConfig;
 import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReference;
 import com.google.gson.Gson;
+import org.springframework.web.server.ResponseStatusException;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
 
 /**
  * When updating the path it should also be updated in the OSGi component.
  */
-@Path("/repair-management/v1")
+@RestController
+@Configuration
+@ComponentScan
```

Comment:
The @ComponentScan/@Configuration annotation does not seem to do anything. The python-tests seems to work without them at least.